src/components/marketing/logo/logo.js

```diff
@@ -2,7 +2,7 @@ import xs from 'xstream'
 import { img } from '@cycle/dom'
 
 const view = () => xs.of(
```

Comment:
We should make master without event logo.
Then do a branch by event

Reply:
i wanted to put alakarte logo, waht do you think ?

Reply:
:ok_hand: